### PR TITLE
fix: just instead make

### PR DESCRIPTION
**Pull request by bestgopher**

### CODE REVIEW

File: docs/src/user/basic_setup.md [hunk 1]
```diff
@@ -70,12 +70,12 @@ This will create a directory named youki in the directory you ran the command in
 
 ## Installing the source
 
-Once you have cloned the source, you can build it using
+Once you have cloned the source, you can build it by [just](https://github.com/casey/just#installation).
```
Comment (YJDoc2) [inline docs/src/user/basic_setup.md:73]:
```suggestion
Once you have cloned the source, you can build it with [just](https://github.com/casey/just#installation) :
```

**Comment by YJDoc2** [comment]
Hey, Thanks for the PR! We missed to change this when switching over to just.
I have left a minor wording suggestion, please take a look!

**Comment by YJDoc2** [comment]
@bestgopher I think the last commit with changes did not have the correct Sign-off, so the DCO check if failing. Can you add the sign-off similar to your first commit?

**Comment by bestgopher** [comment]
> @bestgopher I think the last commit with changes did not have the correct Sign-off, so the DCO check if failing. Can you add the sign-off similar to your first commit?

@YJDoc2 Done.

File: scripts/build.sh
Comment (YJDoc2) [inline scripts/build.sh:1]: Hey please do not change this file with the docs fix. Once we identify the issue in the #2586 , you can make a separate PR.

**Review by YJDoc2 — CHANGES_REQUESTED** [review]

**Review by YJDoc2 — APPROVED** [review]
